src/test/java/it/okkam/validation/FiscalCodeValidatorTest.java

```diff
@@ -78,65 +81,65 @@ public void testDateTimePattern() {
         "1926-03-24T00:00:00", "SANGIANO", "M");
     Assert.assertTrue(arrayContains(codes, "FOXDRA26C24H872Y"));
   }
-  
+
   @Test
   public void testDot() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "FO", "DAR.IO",
         "1926-03-24T00:00:00", "SANGIANO", "M");
     Assert.assertTrue(arrayContains(codes, "FOXDRA26C24H872Y"));
   }
-  
+
   @Test
   public void testBacktick() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "D`AMICO", "ILARIA",
         "1973-08-30T00:00:00", "ROMA", "F");
     Assert.assertTrue(ArrayUtils.contains(codes, "DMCLRI73M70H501N"));
   }
-  
+
   @Test
   public void testAutogenerationOfComuniNamesWhenAccented() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "D'AMICO", "ILARIA",
         "1973-08-30T00:00:00", "ROVERÈ DELLA LUNA", "F");
     Assert.assertTrue(ArrayUtils.contains(codes, "DMCLRI73M70H607G"));
-    
+
     codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "D'AMICO", "ILARIA",
         "1973-08-30T00:00:00", "ROVERE' DELLA LUNA", "F");
     Assert.assertTrue(ArrayUtils.contains(codes, "DMCLRI73M70H607G"));
   }
-  
-  @Test(expected=IllegalArgumentException.class)
+
+  @Test(expected = IllegalArgumentException.class)
   public void testTruncatedComuneNameNotConfigured() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf3, "D'AMICO", "ILARIA",
         "1973-08-30T00:00:00", "MAGRÈ SULLA STRADA DEL VI", "F");
     Assert.assertNotNull(codes);
   }
-  
+
   @Test
   public void testTruncatedComuneName() {
     String[] codes = FiscalCodeValidator.calcoloCodiceFiscale(conf2, "D'AMICO", "ILARIA",
         "1973-08-30T00:00:00", "MAGRÈ SULLA STRADA DEL VINO", "F");
     Assert.assertTrue(ArrayUtils.contains(codes, "DMCLRI73M70E829N"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "DMCLRI73M70E829N" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS1192)